### DISCUSSION TITLE
Experimental Offline Standalone Mac Installer for Stable Diffusion Web UI (unofficial) 20230726

### DISCUSSION THREAD

**Discussion by brkirch** [brkirch/stable-diffusion-webui#24] [Announcements]
### Changelog

- Reduce NansExceptions by using `torch.zeros()` instead of `torch.empty()` for `torch.baddbmm()`. Even though the tensor should be ignored due to `beta=0`, apparently `torch.empty()` can still create problems.
- Go back to using the official k-diffusion repository.
- Use CPU for randn in the UniPC sampler.

<hr /><em>This discussion was created from the release <a href='https://github.com/brkirch/stable-diffusion-webui/releases/tag/20230726_experimental'>Experimental Offline Standalone Mac Installer for Stable Diffusion Web UI (unofficial) 20230726</a>.</em>

**Comment by Max1mu1** [comment]
Thank you but any important extensions still doesn't works, it installed but not appears web ui.
ebsynth_utility
sd-dynamic-prompts
sd-webui-segment-anything

**Reply by brkirch** [reply]
Unfortunately I've seen others reporting extentions also not working with the latest official release, so you may have to wait for the developers of those extensions to update them. I'll take a look at those plugins later and confirm if it is a problem with this repo specifically or not.

**Comment by fckbauer** [comment]
@brkirch  any good news for Roop? I was able to work on the ControlNet, but Roop remains elusive. Any input is appreciated. Cheers!

**Comment by fckbauer** [comment]
also there is a tutorial on how to actually use xformers on Mac M1, just wondering if it will mess up the one that you provided us? Hoping for additional speed boost. I'm on MBP M1 16gb Cheers! @brkirch
